mockgcp/mockbigtable/service.go

```diff
@@ -69,6 +69,13 @@ func (s *MockService) NewHTTPMux(ctx context.Context, conn *grpc.ClientConn) (ht
 		return nil, err
 	}
 
+	// Returns slightly non-standard errors
```

Comment:
I actually don't think these are non-standard errors, I just think we started with the one service that follows a different pattern.  We should address that the default should be the other way round, but not in this PR I suggest.